### PR TITLE
Adding Activity in code

### PR DESCRIPTION
**Pull request by TiagoBrenck**
Included MsalActivity as a replacement of the changes in the manifest.
Updated README.md

### CODE REVIEW

File: UserDetailsClient/UserDetailsClient.Droid/MsalActivity.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿
```
Comment (jennyf19) [inline UserDetailsClient/UserDetailsClient.Droid/MsalActivity.cs]: nit: remove extra line

**Comment by TiagoBrenck** [comment]
@jennyf19 when I tried to update the package, I got this message:
```
Failed to add reference. The package 'Microsoft.Identity.Client' tried to add a framework reference to 'System.Drawing.Common.dll' which was not found in the GAC. This is possibly a bug in the package. Please contact the package owners for assistance.
Reference unavailable.
```

**Comment by jennyf19** [comment]
@TiagoBrenck yes...that's an issue w/packages.config and xamarin. you'll have to manually edit the csproj file to do the update.

**Comment by TiagoBrenck** [comment]
@jennyf19 packages updated!

**Pull request merged**